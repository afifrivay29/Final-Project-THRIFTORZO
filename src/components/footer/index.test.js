import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from './index.js';

it('renders footer correctly', () => {
	const tree = renderer
		.create(
			<MemoryRouter>
				<Footer />
			</MemoryRouter>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
