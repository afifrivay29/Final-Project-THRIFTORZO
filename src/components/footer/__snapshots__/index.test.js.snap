// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  correctly 1`] = `
<div
  className="footer bg-[#7126B5] py-9 hidden md:block"
>
  <div
    className="container"
  >
    <div
      className="ant-row"
      style={
        Object {
          "marginLeft": -12,
          "marginRight": -12,
        }
      }
    >
      <div
        className="ant-col flex md:justify-start justify-center ant-col-xs-24 ant-col-md-12"
        style={
          Object {
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        <span
          className="text-lg font-bold leading-5 text-white md:mb-0 mb-6"
        >
          <img
            alt="Thriftorzo"
            className="h-[56px]"
            src="logo-white.svg"
          />
        </span>
      </div>
      <div
        className="ant-col flex md:justify-end items-center justify-center ant-col-xs-24 ant-col-md-12"
        style={
          Object {
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        <span
          className="md:text-sm text-xs text-white"
        >
          Copyright ©2022 Thriftorzo. All rights reserved.
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders footer correctly 1`] = `
<div
  className="footer bg-[#7126B5] py-9 hidden md:block"
>
  <div
    className="container"
  >
    <div
      className="ant-row"
      style={
        Object {
          "marginLeft": -12,
          "marginRight": -12,
        }
      }
    >
      <div
        className="ant-col flex md:justify-start justify-center ant-col-xs-24 ant-col-md-12"
        style={
          Object {
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        <span
          className="text-lg font-bold leading-5 text-white md:mb-0 mb-6"
        >
          <img
            alt="Thriftorzo"
            className="h-[56px]"
            src="logo-white.svg"
          />
        </span>
      </div>
      <div
        className="ant-col flex md:justify-end items-center justify-center ant-col-xs-24 ant-col-md-12"
        style={
          Object {
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        <span
          className="md:text-sm text-xs text-white"
        >
          Copyright ©2022 Thriftorzo. All rights reserved.
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders product card correctly 1`] = `
<div
  className="footer bg-[#7126B5] py-9 hidden md:block"
>
  <div
    className="container"
  >
    <div
      className="ant-row"
      style={
        Object {
          "marginLeft": -12,
          "marginRight": -12,
        }
      }
    >
      <div
        className="ant-col flex md:justify-start justify-center ant-col-xs-24 ant-col-md-12"
        style={
          Object {
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        <span
          className="text-lg font-bold leading-5 text-white md:mb-0 mb-6"
        >
          <img
            alt="Thriftorzo"
            className="h-[56px]"
            src="logo-white.svg"
          />
        </span>
      </div>
      <div
        className="ant-col flex md:justify-end items-center justify-center ant-col-xs-24 ant-col-md-12"
        style={
          Object {
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        <span
          className="md:text-sm text-xs text-white"
        >
          Copyright ©2022 Thriftorzo. All rights reserved.
        </span>
      </div>
    </div>
  </div>
</div>
`;
